TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -4088,15 +4098,41 @@ private struct BufferItem {
                 return true;
             }
 
-            ExtVehicleType allowedTypes = vehicleRestrictionsManager.GetAllowedVehicleTypes(
+            ExtVehicleType allowedPathVehicleTypes = allowMixedCargoPath_
+                                                        ? mixedPathVehicleTypes_
+                                                        : queueItem_.vehicleType;
+
+            ExtVehicleType allowedLaneVehicleTypes = vehicleRestrictionsManager.GetAllowedVehicleTypes(
```

Comment:
`VehicleRestrictionsManager` does not make use of `LaneType.CargoVehicle` at all. why is that?

Reply:
Silly me. I confused `VehicleRestrictionsManager` with `VehicleBehaviourManager` because I was tired last night.